semanticz-connector-neptune/src/main/java/zone/cogni/semanticz/connectors/neptune/NeptuneRdfStoreService.java

```diff
@@ -0,0 +1,129 @@
+package zone.cogni.semanticz.connectors.neptune;
+
+import org.apache.jena.query.*;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdfconnection.RDFConnection;
+import org.apache.jena.rdfconnection.RDFConnectionRemote;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import zone.cogni.sem.jena.template.JenaResultSetHandler;
+import zone.cogni.semanticz.connectors.general.RdfStoreService;
+
+public class NeptuneRdfStoreService implements RdfStoreService {
+
+    private static final Logger log = LoggerFactory.getLogger(NeptuneRdfStoreService.class);
+
+    private final String sparqlEndpoint;
+    private final String gspEndpoint;
+
+    public NeptuneRdfStoreService(String baseUrl) {
+        if (baseUrl.endsWith("/")) {
+            baseUrl = baseUrl.substring(0, baseUrl.length() - 1);
+        }
+        this.sparqlEndpoint = baseUrl + "/sparql";
+        this.gspEndpoint = baseUrl + "/sparql/gsp";
+    }
+
+    private RDFConnection getConnection() {
+        return RDFConnectionRemote.newBuilder()
+                                  .destination(sparqlEndpoint)
+                                  .queryEndpoint(sparqlEndpoint)
+                                  .updateEndpoint(sparqlEndpoint)
+                                  .gspEndpoint(gspEndpoint)
+                                  .build();
+    }
+
+    private Query buildQuery(Query query, QuerySolutionMap bindings) {
```

Comment:
little strange. this location. usually called methods are below calling methods (at least with better compiler technologie vs 80s 😄 )